anymail/backends/scaleway.py

```diff
@@ -0,0 +1,163 @@
+from ..exceptions import AnymailAPIError, AnymailConfigurationError
+from ..message import AnymailRecipientStatus
+from ..utils import get_anymail_setting
+from .base_requests import AnymailRequestsBackend, RequestsPayload
+
+
+class EmailBackend(AnymailRequestsBackend):
+    """
+    Scaleway Transactional Email API Backend
+    """
+
+    esp_name = "Scaleway"
+
+    def __init__(self, **kwargs):
+        """Init options from Django settings"""
+        esp_name = self.esp_name
+        self.api_key = get_anymail_setting(
+            "api_key", esp_name=esp_name, kwargs=kwargs, allow_bare=True
+        )
+        if not self.api_key:
+            raise AnymailConfigurationError(
+                "You must set the ANYMAIL_SCALEWAY_API_KEY setting."
+            )
+
+        self.project_id = get_anymail_setting(
+            "project_id", esp_name=esp_name, kwargs=kwargs, default=None
+        )
+        self.region = get_anymail_setting(
+            "region", esp_name=esp_name, kwargs=kwargs, default="fr-par"
+        )
+
+        api_url = get_anymail_setting(
+            "api_url",
+            esp_name=esp_name,
+            kwargs=kwargs,
+            default=(
+                "https://api.scaleway.com/transactional-email/v1alpha1/regions/"
+                f"{self.region}"
+            ),
+        )
+        if not api_url.endswith("/"):
+            api_url += "/"
+        super().__init__(api_url, **kwargs)
+
+    def build_message_payload(self, message, defaults):
+        return ScalewayPayload(message, defaults, self)
+
+    def parse_recipient_status(self, response, payload, message):
+        parsed_response = self.deserialize_json_response(response, payload, message)
+        statuses = {}
+        try:
+            emails = parsed_response["emails"]
+        except (KeyError, TypeError):
+            raise AnymailAPIError(
+                "Invalid response from Scaleway API",
+                email_message=message,
+                payload=payload,
+                response=response,
+                backend=self,
+            )
+
+        for email_info in emails:
+            recipient = email_info.get("mail_rcpt")
+            message_id = email_info.get("id")
```

Comment:
(Note to self: using the response's `"id"` field as Anymail's `message_id` is correct ✔️. Scaleway describes it as the message's "technical id," and it matches the `"email_id"` provided in the tracking [webhook payload](https://www.scaleway.com/en/docs/transactional-email/reference-content/webhook-events-payloads/). Scaleway also provides a `"message_id"` field which is a portion of the *Message-Id* header sent with the email, but that value seems to be useless for webhooks or calling other Scaleway APIs like "get email".)